src/IssueViz/Commands/IssueVizPackage.vsct

```diff
@@ -56,6 +56,12 @@
 
   </Commands>
 
+  <CommandPlacements>
+    <CommandPlacement guid="guidIssueVizPackageCmdSet" id="IssueVisualizationToolWindowCommandId" priority="0x200">
```

Comment:
Is this going to be visible for every type of issue, or just SonarLint issues? I don't think it should appear for all issues - that would just be confusing for the user.

Reply:
Hm, I don't see an option to specify a context for a command placement. There are [conditions](https://docs.microsoft.com/en-us/visualstudio/extensibility/vsct-xml-schema-conditional-attributes?view=vs-2019) but I don't think it's the same. I'll check it.

Reply:
Have a look at the existing "disable SonarLint rule" command. It checks whether the selected error is a Sonar error or not.
In this case, we might also want to set the enabled status based on whether issue has secondary locations or not.

It would be simpler to separate the changes: the first two commits are simple and are related, so commit them now, and work on the "add to context menu" separately since it's logically separate, needs more work, and it's not obvious what the UX should be.